targets/XPlatCppSdk/source/test/TestApp/PlayFabApiTest.cpp

```diff
@@ -255,12 +255,13 @@ namespace PlayFabUnit
 
         time_t now = time(nullptr);
         struct tm timeinfo;
-#if defined(PLAYFAB_PLATFORM_IOS) || defined(PLAYFAB_PLATFORM_ANDROID)
+#if defined(PLAYFAB_PLATFORM_IOS) || defined(PLAYFAB_PLATFORM_ANDROID) || defined(PLAYFAB_PLATFORM_LINUX)
         timeinfo = *gmtime(&now);
-#else // PLAYFAB_PLATFORM_IOS || PLAYFAB_PLATFORM_ANDROID
+        now = timegm(&timeinfo);
```

Comment:
>now = timegm(&timeinfo); [](start = 8, length = 24)

You may break already tested Android and iOS builds with this change. Please note, that before you change on Android and iOS it was calling:

now = _mkgmtime(&timeinfo);

Now, after your change, it will be calling:

now = timegm(&timeinfo);

not only for Linux, but also for Android and iOS. This seems dangerous to me. Did you test on Android and iOS if your change works? :) Probably not. Therefore, i would modify your code in such a way, that timegm(&timeinfo) is called only on Linux. For Android and iOS it should remain how it was.